doc/config-lnd-integrated.md

```diff
@@ -117,6 +119,15 @@ system:
 - **On Linux**: `~/.lit/lit.conf`
 - **On Windows**: `~/AppData/Roaming/Lit/lit.conf`
 
+## LiT and LND interfaces
+
+Port 10009 is the port that LND uses to expose its gRPC interface. LND's tls 
+cert and macaroons will be required when making requests to this interface.
+
+Port 8443 is a port that LiT uses to expose a variety of interfaces: gRPC, 
+REST, grpc-web. When making requests using this interface, LiT's tls cert and 
```

Comment:
We have example commands for lncli/loop/pool/faraday below, should we also add an example command for `litcli`? Here and in the remote mode docs?

Reply:
I'll create a new PR for this